hr_recruitment_skill/models/hr_job.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class HrJob(models.Model):
+    _inherit = 'hr.job'
+
+    required_skill_ids = fields.Many2many(
+        'hr.skill',
+        relation='hr_jobs_required_skills',
```

Comment:
Are you sure about this relation's name? I couldn't find it in the dependencies.